magpylib/_lib/obj_classes/class_Sensor.py

```diff
@@ -11,31 +11,92 @@
 # ON INTERFACE
 class Sensor(BaseGeo, BaseDisplayRepr):
     """
-    Magnetic field sensor. Can be used as observer input.
+    Magnetic field sensor. Can be used as observer input for magnetic field
+    computation.
 
-    Reference position: Origin of the local CS of the sensor.
-
-    Reference orientation: Local and global CS coincide at initialization.
+    Local object coordinates: Sensor pixel (=sensing elements) are defined in the local
```

Comment:
`(=sensing elements)` prolly a typo.

Reply:
yeah ... wasnt a typo, but I can see how the "=" is not necessary :)